fennel/lib/last/last.py

```diff
@@ -19,7 +19,7 @@
 # ------------------------------------------------------------------------------
 
 
-class Window(BaseModel):
+class Last(BaseModel):
```

Comment:
Having `Last` is great, but I don't think we can remove old `Window` just like that - else it's a breaking change for existing users?

I'd suggest keeping old `Window` around for now (it will have different path from new `Window` so won't collide) + all documentation can change to using either `Last` or no constructor. 

The next time we do a breaking change (say in `1.0.0` launch), we can then remove the old `Window`. cc @aditya-nambiar for thoughts.

Reply:
Removing window will be a breaking change.

I was thinking along the same lines, anyhow this change has to be a major version too, for now I will add the window, but I will change the docs to remove any mention wither Window or Last.

Reply:
Sounds good. Yeah, we are planning a `1.0` alpha release at the end of Q1A - we could make a batch of breaking changes to the API in that one.